### PR TITLE
WV-287 convert methods with warning to @static method TEAM REVIEW

### PR DESCRIPTION
**Pull request by tejasaditya**
Removed self parameter for functions with warning and converted to @staticmethod.

Also fixed a PEP 8: W391 blank line at end of file

### CODE REVIEW

File: party/models.py [hunk 1]
```diff
@@ -102,4 +104,3 @@ def retrieve_all_party_names_and_ids(self):
             handle_record_not_found_exception(e, logger=logger)
 
         return party_items_list
```
Comment (DaleMcGrew) [inline party/models.py:106]: Hi @tejasaditya I would like to maintain the standard of a new line at the end of the file. I'm curious if your IDE gave you a warning to remove this linebreak?

Comment (tejasaditya) [reply party/models.py:106]: Hi @DaleMcGrew. It looks like my IDE does have line 107 as a blank line but it doesn't match what I see in the repository online. PyCharm did warn me about having a line so I added it. I also have no unstaged/uncommitted changes

Comment (DaleMcGrew) [reply party/models.py:106]: My apologies, I read the .github diff wrong.

**Comment by DaleMcGrew** [comment]
👍

**Pull request merged**